onnxruntime/core/providers/nuphar/scripts/symbolic_shape_infer.py

```diff
@@ -1038,6 +1041,38 @@ def _infer_Slice(self, node):
             assert len(ends) == 1
             self.sympy_data_[node.output[0]] = self.sympy_data_[node.input[0]][starts[0]:ends[0]]
 
+    def _infer_Gelu(self, node):
+        data_shape = self._get_shape(node, 0)
+        vi = self.known_vi_[node.output[0]]
+        vi.CopyFrom(helper.make_tensor_value_info(node.output[0], onnx.TensorProto.FLOAT, data_shape))
+        self.known_vi_[vi.name] = vi
```

Comment:
>self.known_vi_[vi.name] = vi [](start = 8, length = 28)

seems not needed? vi.name is node.output[0], which matches when vi is assigned from self.known_vi_